docs/Python-Intro.rst

```diff
@@ -195,7 +195,7 @@ Early stopping requires at least one set in ``valid_sets``. If there is more tha
 The model will train until the validation score stops improving.
 Validation score needs to improve at least every ``early_stopping_rounds`` to continue training.
 
-If early stopping occurs, the model will have an additional field: ``bst.best_iteration``.
+If early stopping occurs, the iteration index will be saved in the ``bst.best_iteration`` field.
```

Comment:
It seems, as long as set the early_stopping_rounds, the best_iteration will be set whatever early stopping occurs or not.